AdminWebsite/AdminWebsite.UnitTests/Extensions/HearingDetailsResponseExtensionsTests.cs

```diff
@@ -69,79 +69,79 @@ public void Should_Return_False_If_Judge_Email_is_null()
 
             _hearing.HasJudgeEmailChanged(hearing2).Should().BeFalse();
         }
-        
+
         [Test]
         public void Should_Return_True_If_Judge_Has_Changed_When_Comparing_Judge_Emails()
         {
             var otherInfo = new OtherInformationDetails { JudgeEmail = "judge@hmcts.net" };
             _hearing.OtherInformation = otherInfo.ToOtherInformationString();
 
-            var hearing2 = new HearingDetailsResponse {Id = Guid.NewGuid(), Participants = new List<ParticipantResponse>()};
+            var hearing2 = new HearingDetailsResponse { Id = Guid.NewGuid(), Participants = new List<ParticipantResponse>() };
             var hearing2OtherInfo = new OtherInformationDetails { JudgeEmail = "judge2@hmcts.net" };
             hearing2.OtherInformation = hearing2OtherInfo.ToOtherInformationString();
-            
+
             _hearing.HasJudgeEmailChanged(hearing2).Should().BeTrue();
         }
-        
+
         [Test]
         public void should_return_true_when_ejud_email_has_been_assigned_from_no_judge()
         {
             var judge = Builder<ParticipantResponse>.CreateNew()
                 .With(x => x.UserRoleName = UserRole.Judge.ToString())
-                .With(x => x.ContactEmail = "new@judiciaryejud.com")
+                .With(x => x.Username = "new@judiciaryejud.com")
                 .Build();
             var newHearing = new HearingDetailsResponse
             {
                 Id = Guid.NewGuid(),
-                Participants = new List<ParticipantResponse> {judge}
+                Participants = new List<ParticipantResponse> { judge }
             };
             _hearing.HasJudgeEmailChanged(newHearing).Should().BeTrue();
         }
-        
+
         [Test]
         public void should_return_true_when_ejud_email_has_changed_to_another_ejud()
         {
             var existingEJudJudge = Builder<ParticipantResponse>.CreateNew()
                 .With(x => x.UserRoleName = UserRole.Judge.ToString())
-                .With(x => x.ContactEmail = "old@judiciaryejud.com")
+                .With(x => x.Username = "old@judiciaryejud.com")
                 .Build();
             _hearing.Participants.Add(existingEJudJudge);
-            
+
             var newEJudJudge = Builder<ParticipantResponse>.CreateNew()
                 .With(x => x.UserRoleName = UserRole.Judge.ToString())
                 .With(x => x.ContactEmail = "new@judiciaryejud.com")
                 .Build();
             var newHearing = new HearingDetailsResponse
             {
                 Id = Guid.NewGuid(),
-                Participants = new List<ParticipantResponse> {newEJudJudge}
+                Participants = new List<ParticipantResponse> { newEJudJudge }
             };
             _hearing.HasJudgeEmailChanged(newHearing).Should().BeTrue();
         }
-        
+
         [Test]
         public void should_return_true_when_ejud_email_has_changed_to_vh_judge()
```

Comment:
Consider using Pascal Case, for test names, eg:
ShouldReturnTrue_WhenEjudEmailHasChangedToVHJudge helps to improve readability or 
MethodUnderTest_ShouldReturnTrue_WhenEjudEmailHasChangedToVHJudge could also be added for additional context when test is viewed in isolation.

Reply:
these tests were inherited, we will write in the pascal case.